lib/modules/subredditManager.js

```diff
@@ -438,6 +438,8 @@ function showSubredditGroupDropdown(obj) {
 }
 
 function _showSubredditGroupDropdown(obj, subreddits, suffix) {
+	// Updates source object referenced on the 'click' events
+	subredditGroupDropdownRefItem = obj;
```

Comment:
I didn't like it either, but I didn't see an easy solution, so I went with essentially how everything else was working and just copied that.